content/faq/publishing-standards.md

```diff
@@ -0,0 +1,21 @@
+---
+title: What are the recommended dimensions for thumbnails and covers?
+category: publisher
+order: 2
+---
+
+Even though LBRY does not enforce any file size, dimension, or aspect ratio properties, our applications are optimized to provide the best experience for viewing content thumbnails, channel covers, and channel profile thumbnails. If you are using the apps to upload the images, these uploads go through [spee.ch](https://spee.ch) which is limited to 10MBs per file. 
+
+### Supported file types
+A majority of image file types including jpg, png, and gif can be hyperlinked or uploaded. 
```

Comment:
Standard and common image types like jpg, png, and gif are all supported and recommended. Additional image types that render in most modern web-browsers can be used, but are less common and tested.